glide.lock

```diff
@@ -62,6 +85,25 @@ imports:
   version: 0208149b40d863d2c1a2f8fe5753096a9cf2cc8b
 - name: github.com/Sirupsen/logrus
```

Comment:
hmm, so  I need to implement a logger interface to do this. I was also thinking to change architecture a little bit. Well I will made a separate PR for this.